README.md

```diff
@@ -27,9 +32,19 @@ mkdir myApp && cd myApp
 yo react-zeal
 
 # install all the things
+
+# with yarn
+yarn
+
+# with npm
 npm install
 
 # start your app on http://localhost:3000
+
+# with yarn
+yarn start
+
+# with npm
```

Comment:
I think we're indicating we prefer yarn by putting it first, but to me it isn't something we want to push. I'd suggest that our goal is to provide exactly the information necessary to install and start the app, regardless of what tools a user prefers.